src/app/core/models/user-settings.model.ts

```diff
@@ -25,7 +25,7 @@ export interface UserSetting extends InfoPopupSettings {
   featureToggeGpsDebug: boolean;
   copyright?: string;
   photographer?: string;
-  lastOverridden?: Date;
+  lastOverridden?: number;
```

Comment:
Bruk gjerne JsDoc for å dokumentere denne også, slik vi har gjort for de nye innstillingene under